src/layouts/EditContactUs.jsx

```diff
@@ -774,27 +783,49 @@ const EditContactUs = ({ match }) => {
 
   return (
     <>
-      {showDeletedText && !isEmpty(deletedFrontMatter) && (
+      {!isEmpty(deletedFrontMatter) && (
         <GenericWarningModal
+          isOpen={isRemovedOpen}
```

Comment:
we should omit the `!isEmpty(deletedFrontMatter)` and inline this into the `isOpen` prop; separately, we might wanna double check if this works as intended.

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/513eedac744b2fe3c2814aff9b935982147c46d2